backend/routers/index_router.py

```diff
@@ -25,5 +30,18 @@ async def read_main():
 
 @index_router.get("/get_pixel_watch", response_model=List[schemas.PixelWatch])
 def get_pixel_watch(db: Session = Depends(get_db)):
-    users = db.query(models.PixelWatch).all()
+    users = db.query(PixelWatch).all()
     return users
+
+class WatchPixelParams(BaseModel):
+    lat: float
+    lon: float
+    date: datetime
+
+@index_router.post("/watch_my_pixel")
+async def watch_my_pixel(params: WatchPixelParams,
+                         user: UserResponse = Depends(get_current_user),
+                         db: Session = Depends(get_db)):
+    db.add(PixelWatch(latitude=params.lat, longitude=params.lon, datetime=params.date, user_id=user.user.id))
```

Comment:
Consider storing path & rów. Because it will make it eaisier  for us to detect who subscribed for specific Landsat scene

Reply:
bro storing rów, also it should work but only in theory